### ISSUE TITLE
Documentation for testharness.js

### ISSUE THREAD

**Issue by tobie** [w3c/testtwf-website#19]
E.g. based Dzenana Trenutak's [work](http://dzenana-trenutak.github.com/GitDocs/W3C_HTMLTestsuite_Tutorial/index.html).

**Comment by marcoscaceres** [comment]
This one has worked for me: http://darobin.github.io/test-harness-tutorial/docs/using-testharness.html

Could be adapted to fit better the goals of the project.

**Comment by tobie** [comment]
Yes. We should also extract the documentation from the comment src code itself.

**Comment by marcoscaceres** [comment]
a single source for that information would be good (either directly coming from the source code or from a published doc). I think a published document would be better - it's easier to search, etc.... but others doing actual testing might have a different opinion on that. 

is there a plan to move testharness.js to GH?

**Comment by tobie** [comment]
testharness.js [is on GH](https://github.com/w3c/testharness.js). Documentation is in the [src itself](https://github.com/w3c/testharness.js/blob/master/testharness.js), which is a real shame.

**Comment by marcoscaceres** [comment]
Right, @jgraham made fun of me on IRC for not knowing that :) The project page also points to [Robin's document](http://darobin.github.io/test-harness-tutorial/docs/using-testharness.html) as its documentation.

**Comment by tobie** [comment]
Points to @darobin's doc and the src code.

Given the goal of this effort is to reconcile documentation, we might want to include both of these here, no?

**Comment by jgraham** [comment]
Yes, in particular darobin's doc.

The nice thing about documentation in the source code is that it is very easy to keep up to date. But maybe the pace of development has slowed enough that isn't a significant consideration anymore.

**Comment by tobie** [comment]
:+1:

**Comment by darobin** [comment]
Yes, please take my document! It's a bit of a pain to produce (docco relies on some Python thing that seems to be broken every time I update it) but it probably doesn't need that much maintenance.

**Comment by marcoscaceres** [comment]
What's the standard format for documents produced by the group?

**Comment by tobie** [comment]
> What's the standard format for documents produced by the group?

Markdown. More details [here](https://github.com/w3c/ttwf-docs/blob/gh-pages/CONTRIBUTING.md).

**Comment by hubenjamin** [comment]
Thanks for all of the comment and sources. I am compiling both Robin's doc and extracting from source for the doc.

**Comment by hubenjamin** [comment]
I submitted a pull request for 2 docs:

testharness-documentation.md - extracted from the src

testharness-tutorial.md - Robin's doc. Note that I removed 2 paragraphs at introduction, and the Result section at the end. One paragraph about how this documentation is generated, which is not applicable any more. Another one about runnable code and test Result at the end after loading this doc, which is also not applicable any more.
